src/app/submission/submission-direct/direct-submit-file-upload.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { Observable } from "rxjs";
+import { FileService } from '../../file/shared/file.service';
+import { Path } from '../../file/shared/path';
+import { UploadEvent } from '../../file/shared/http-upload-client.service';
+
+@Injectable()
+export class DirectSubmitFileUploadService {
+  private fileService: FileService;
+  private path: Path = new Path('/User', '/');
```

Comment:
I'm not sure if you need to consider groups for this scenario

Reply:
No, all the files are going to be uploaded to the `Home` group.